### PR TITLE
Update Hackathon index.md

### PR DESCRIPTION
**Pull request by MadMax93**
Added Hackathon information for 22.10.2024 and updated process plugin versions

### CODE REVIEW

File: docs/src/hackathon/index.md [hunk 1]
```diff
@@ -11,40 +11,51 @@ icon: guide
 :::
 
 
-# Hackathon vom 18.06.2024
+# Hackathon vom 22.10.2024
 
 ::: tip Zusammenfassung
-- Konferenz: https://audimax.heiconf.uni-heidelberg.de/jxh4-jxx2-tm6c-d37q
+- Konferenz:
```
Comment (schwzr) [inline docs/src/hackathon/index.md:17]: delete line?

File: docs/src/hackathon/index.md [hunk 2]
```diff
@@ -11,40 +11,51 @@ icon: guide
...
-   - Data Transfer
-   - KDS Report
-   - Blaze
-   - Flare mit neuer KDS Ontologie
-3. Testen des Prozessplugins "Data Sharing" (ab 14:15 Uhr):
-   - Praktische Anwendung des "Data Sharing"-Prozessplugins auf unserer Testinfrastruktur.
+Wie üblich sollen die Tools und Prozesse hands-on auf der Testinfrastruktur erprobt und deployed werden, dazu folgen weitere Informationen kurz vor dem Hackathon.
 
-Ziel der Veranstaltung ist es die Test- und Produktivsysteme möglichst aller Standorte auf den neusten Stand zu patchen.
+Vorläufiger Zeitplan: Link
```
Comment (schwzr) [inline docs/src/hackathon/index.md:31]: remove "Link"?

File: docs/src/hackathon/index.md [hunk 3]
```diff
@@ -11,40 +11,51 @@ icon: guide
...
+
+Viele Grüße,
+Julian Gründner und Maximilian Kurscheidt
+für FDPG+ und DSF Community
 
 
 
 ## Vorbereitung
 In diesem Hackathon ist das Aktualisieren der Systeme ein aktiver Bestandteil und bedarf keiner speziellen Vorbereitung. Für das Testen der Data Sharing Prozesse sollten jedoch einige Voraussetzungen bereits vor dem Termin erfüllt sein.
+###TBD
```
Comment (schwzr) [inline docs/src/hackathon/index.md:58]: ?

File: docs/src/hackathon/index.md [hunk 4]
```diff
@@ -72,14 +83,16 @@ Eine Installation kann vor oder während des Hackathons durchgeführt werden.
 ### Prozessplugins
```
Comment (schwzr) [inline docs/src/hackathon/index.md:83]: add note that versions are subject to change and new releases of some of the plugins are expected

File: docs/src/hackathon/index.md [hunk 5]
```diff
@@ -72,14 +83,16 @@ Eine Installation kann vor oder während des Hackathons durchgeführt werden.
...
+- MII Feasibility 1.0.0.6 [Instructions](https://github.com/medizininformatik-initiative/feasibility-deploy/wiki/DSF-Middleware-Setup)
+- MII Process Report 1.1.1.1 for Test Infrastructure [Instructions](https://github.com/medizininformatik-initiative/mii-process-report/wiki/Process-Report-Deployment-v1.1.x.x). Please use 1.1.1.1 for Prod for now [Instructions](https://github.com/medizininformatik-initiative/mii-process-report/wiki/Process-Report-Deployment-v1.1.x.x)
+- MII Process Data Transfer 1.0.2.1 [Instructions](https://github.com/medizininformatik-initiative/mii-process-data-transfer/wiki/Process-Data-Transfer-Deployment-v1.0.x.x)
+- MII Process Data Sharing 1.0.1.1 [Instructions](https://github.com/medizininformatik-initiative/mii-process-data-sharing/wiki/Process-Data-Sharing-Deployment-v1.0.x.x)
 
 ### Weitere Anwendungen
 - Flare [Instructions](https://github.com/medizininformatik-initiative/flare)
-- Blaze 0.27.1 [Instructions](https://github.com/samply/blaze/blob/master/docs/deployment/README.md)
+- Blaze 0.30.0 [Instructions](https://github.com/samply/blaze/blob/master/docs/deployment/README.md)
+- FHIR Evaluator [Instructions]()
```
Comment (schwzr) [inline docs/src/hackathon/index.md:94]: link missing

File: docs/src/hackathon/index.md [hunk 6]
```diff
@@ -72,14 +83,16 @@ Eine Installation kann vor oder während des Hackathons durchgeführt werden.
...
+- TORCH [Instructions]()
```
Comment (schwzr) [inline docs/src/hackathon/index.md:95]: link missing

**Review by schwzr — CHANGES_REQUESTED** [review]

File: docs/src/hackathon/index.md (hunk 3 above)
Comment (MadMax93) [reply docs/src/hackathon/index.md:58]: Some things are still missing from FPDG+. But I can remove that.

**Review by schwzr — APPROVED** [review]

**Pull request merged**